apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -229,6 +229,17 @@ function load_blog_posts_block() {
  * Load WPCOM Block Editor NUX
  */
 function load_wpcom_block_editor_nux() {
+	/**
+	 * Can be used to enable Dotcom editor guide.
+	 *
+	 * @since 0.23
+	 *
+	 * @param bool true if Dotcom editor nux should be enabled, false otherwise.
+	 */
+	if ( apply_filters( 'a8c_enable_dotcom_editor_nux', false ) ) {
```

Comment:
Do we need to flip the logic on this, early return if this `apply_filters` returns false?

Reply:
Yes, just updated it.